crates/engine/src/rocksdb_engine/mod.rs

```diff
@@ -104,6 +111,17 @@ impl RocksEngine {
         Ok(size)
     }
 
+    /// Gets the max write data size.
+    /// Max write data size = 2 * key + value + `cf_handle_size` + `ESTIMATE_WRITTEN_SIZE_OFFSET`
+    pub(super) fn max_write_size(table_len: usize, key_len: usize, value_len: usize) -> usize {
+        const ESTIMATE_WRITTEN_SIZE_OFFSET: usize = 1008;
```

Comment:
Please leave a comment for this constant.

Reply:
fixed